File: app/src/main/java/com/ashlikun/okhttputils/simple/GoodListData.java
package com.ashlikun.okhttputils.simple;

import com.ashlikun.okhttputils.http.response.HttpResponse;

import java.util.List;

/**
 * 作者　　: 李坤
 * 创建时间: 2018/5/10 0010　下午 3:16
 * 邮箱　　：496546144@qq.com
 * <p>
 * 功能介绍：
 */
public class GoodListData extends HttpResponse {

    public int currentPage;
    public int pageCount;
    public int currentRows;
    public List<ListBean> data;


    public static class ListBean {
        public String goods_id;
        public String goods_name;
        public String type;
        public String Price;
        public String wlPrice;
        public String scPrice;
        public String default_image;
        public String seller_id;
        public int sale;
        public String goodsNowStock;
        public String good_rate;
        public String has_storecoupon;
        public String act_image;
    }
}

File: app/src/main/java/com/ashlikun/okhttputils/simple/data/GoodListData.kt
package com.ashlikun.okhttputils.simple.data

import com.ashlikun.okhttputils.http.response.HttpResponse

/**
 * 作者　　: 李坤
 * 创建时间: 2018/5/10 0010　下午 3:16
 * 邮箱　　：496546144@qq.com
 *
 *
 * 功能介绍：
 */
class GoodListData : HttpResponse() {
    var currentPage = 0
    var pageCount = 0
    var currentRows = 0
    var list: List<ListBean>? = null

    class ListBean {
        var title = 0
        var goods_id: String? = null
        var goods_name: String? = null
        var type: String? = null
        var Price: String? = null
        var wlPrice: String? = null
        var scPrice: String? = null
        var default_image: String? = null
        var seller_id: String? = null
        var sale = 0
        var goodsNowStock: String? = null
        var good_rate: String? = null
        var has_storecoupon: String? = null
        var act_image: String? = null
    }
}